### ISSUE TITLE
Directives broken in 4.0.1 w/ Angular 5

### ISSUE THREAD

**Issue by DutchmanDavid** [swimlane/ngx-dnd#80]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Sorry, we will not be able to answer every support request.  Please consider other venues for support requests
```

**Current behavior**
I have a small list of strings to test this package.
When I drag one string above the other, it disappears! If I print the `orderableLists` to the console, the string that disappeared has become `undefined`.

**Expected behavior**
For the string to move to the correct position, and not disappear.

**Reproduction of the problem**
I have, in angular 5, in my .ts:

    orderableLists = 
        [
          "string1",
          "string2",
          "string3",
          "string4",
          "string5"
        ];

and in my .html:

    <div class="ngx-dnd-container" ngxDroppable [model]="orderableLists">
        <div class="ngx-dnd-item" ngxDraggable *ngFor="let item of orderableLists">
          {{item}}
        </div>
    </div>

Strangely, the next example works just fine:

    <ngx-dnd-container
      [model]="orderableLists">
    </ngx-dnd-container>

as does:

    <div class="ngx-dnd-container" ngxDroppable>
      <div class="ngx-dnd-item" ngxDraggable>Item 1</div>
      <div class="ngx-dnd-item" ngxDraggable>Item 2</div>
      <div class="ngx-dnd-item" ngxDraggable>Item 3</div>
    </div>

**Please tell us about your environment:** VS2017, Angular 5, Windows 10 (1803)
* **ngx-charts version:** 4.0.1 (ngx-charts? isn't this for ngx-dnd? 😄)
* **Angular version:** 5.2.10
* **Browser:** [Chrome 66, Chrome Beta 67]
* **Language:** [TypeScript 2.6]

**Comment by Hypercubed** [comment]
It's going to be pretty had to keep backwords compat on this.  Are you able to upgrade to angular 6.x or drop ngx-dnd down to 3.x?

**Comment by DutchmanDavid** [comment]
I've tried that, but npm didn't want to downgrade it (right before the end of the work day). I'll try again tomorrow morning and give you an update then, but first some sleep :)

**Comment by DutchmanDavid** [comment]
I've tested it on a new Angular 5 project made via ng. No dice.

This time instead of using VS2017 I've used Visual Studio Code with ngx-dnd v4.0.1, v3.2.0 and v3.0.0. All of them don't work.

This is my HTML in app.component.html:

    <h1>ngx container</h1>
    <ngx-dnd-container [model]="orderableLists">
    </ngx-dnd-container>
    
    <h1>divs</h1>
    <div class="ngx-dnd-container" ngxDroppable [model]="orderableLists">
      <div class="ngx-dnd-item" ngxDraggable *ngFor="let item of orderableLists">{{item}}</div>
    </div>

With this in app.component.ts"

    orderableLists = [
      'Item 1a',
      'Item 2a',
      'Item 3a'
    ];

So no crazy code that could influence ngx-dnd. No errors in Chrome's Console either.

----

I even updated Angular from 5 to 6 (with ngx-dnd back to 4.0.1). Still nothing.

Here's the visual output from the HTML above after I dragged an item from the **divs** list around (ngx container list still works fine):
https://imgur.com/3FvThBc
